pybaseball/enums/fangraphs/month.py

```diff
@@ -0,0 +1,12 @@
+from enum import Enum, unique
+
+
+@unique
+class FanGraphsMonth(Enum):
+    ALL               = 0
+    MARCH_APRIL       = 4
```

Comment:
ah, yes, my birth month, the month of March-April.
(I'm aware it's like this there, I'm just laughing at it)